artifactory/cli.go

```diff
@@ -4020,13 +4025,18 @@ func createConfigCommandConfiguration(c *cli.Context) (configCommandConfiguratio
 	}
 	configCommandConfiguration.EncPassword = c.BoolT("enc-password")
 	configCommandConfiguration.Interactive = cliutils.GetInteractiveValue(c)
+	configCommandConfiguration.BasicAuthOnly = c.Bool("basic-auth-only")
 	return
 }
 
 func validateConfigFlags(configCommandConfiguration *commands.ConfigCommandConfiguration) error {
 	if !configCommandConfiguration.Interactive && configCommandConfiguration.ArtDetails.Url == "" {
 		return errors.New("the --url option is mandatory when the --interactive option is set to false or the CI environment variable is set to true.")
 	}
+	// Validate the option is not used along with an access token
+	if configCommandConfiguration.BasicAuthOnly && configCommandConfiguration.ArtDetails.AccessToken != ""{
+		return errors.New("the --basic-auth-only option is only supported with basic authentication options")
```

Comment:
...only supported with basic authentication options --> ...only supported when username and password / API key are provided.